src/platform/user/profile/vet360/tests/components/Transaction.unit.spec.jsx

```diff
@@ -47,15 +56,14 @@ describe('<Vet360Transaction/>', () => {
         },
       },
     });
-
     expect(
-      component.find('Vet360TransactionPending'),
+      component.find(Vet360TransactionPending),
       'renders a transaction-pending message',
     ).to.have.lengthOf(1);
     expect(
-      component.html(),
-      'does not render children components when there is a pending transaction',
-    ).to.not.contain('Children');
+      component.find('div.content'),
```

Comment:
we can't call `html()` anymore since the Transaction component will try to render out the Redux-connected TransactionPending component. But we can use `find()` instead.